Uslov.Operator.Task4/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Uslov.Operator.Task4
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите x ");
+            var x = double.Parse(Console.ReadLine());
+            if(x<-3)
+            {
+                Console.WriteLine("f(x)=0");
+            }
+
+            else if(x>=-3 & x<=-1)
```

Comment:
Лишнее логическое выражение, ведь мы попадаем в эту ветку только тогда, когда предыдущее логическое условие ложно, т. е. когда _x_ >= -3.
```suggestion
            else if(x<=-1)
```